.gitignore

```diff
@@ -68,3 +68,4 @@ fastlane/screenshots
 fastlane/test_output
 .env
 .ruby-version
+.DS_Store
```

Comment:
Just a heads up, this belongs in your system `$HOME/.gitconfig` not the project.